docs/Available-Symbols-Generators.md

```diff
@@ -47,7 +47,9 @@ Available built-in generators for computing generated symbols values are:
 | [now](#now) | Get the current date/time. |
 | [random](#random) | Generate random int. |
 | [regex](#regex) | Process a regular expression. |
+| [regexMatch](#regexmatch) | Process a regular expression with matching. |
```

Comment:
nit: imo this description is a bit confusing, probably "checks if the value matches the regex pattern" is more clear. 
Current wording might be confused with conditional regex processing.